docs/05_components/doc-management/doc-management-server/01_configuring-file-manager.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Document Management - Configuring File Storage Manager'
+sidebar_label: 'Configuring File Storage Manager'
+id: doc-management-configuring
+keywords: [doc-management, file-storage-manager]
+tags:
+  - doc-management
+  - file-storage-manager
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In order to start using Document Management, you must specify your `STORAGE_STRATEGY` in your `system-definition.kts`.
+
+`STORAGE_STRATEGY` supports the following values:
+- LOCAL
+- AWS
+- SHAREPOINT
+- SHAREPOINT_ON_PREM
+
+```kotlin
+item(name = "STORAGE_STRATEGY", value = "<strategy>")
+```
+Once you have set you `STORAGE_STRATEGY`, you must set your storage solution specify configuration.
+
+## Local File Storage
```

Comment:
I got a bit confusing reading this. If the folder is created automatically, why do I need to specify this item? Does it create a new folder with "<folderName>" under /genesis-home/runtime/fileupload/?

Reply:
When no value is passed, the framework will default to "genesis-home/runtime/fileupload/LOCAL_STORAGE". I have pushed the change to make this clearer 👍